### PR TITLE
Fix: paleontologist in fossil excavator

### PR DESCRIPTION
**Pull request by hannibal002**
## What
Paleontologist I book now gets detected via this workaround

### to reproduce:
put
`LorenzChatEvent("    §r§fEnchanted Book", ChatComponentText("    §r§fEnchanted Book")).postAndCatch()`
below
`endPattern.matches(message)`

## Changelog Fixes
+ Fixed Paleontologist book not getting detected when excavate fossils. - hannibal2

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt [hunk 1]
```diff
@@ -101,10 +101,14 @@ object FossilExcavatorAPI {
             inLoot = false
             return
         }
-
-        val pair = itemPattern.matchMatcher(message) {
-            ItemUtils.readItemAmount(group("item"))
-        } ?: return
+        // Workaround: If it is a enchanted book, we assume it is a paleontologist I book
+        val pair = if (message == "    §r§fEnchanted Book") {
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt]: i think someone in the discord said they got more than one book in the same excavation, this wont catch that

Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt]: crazy rare chance, also do we know how that chat message looks like?
just the gray x1?

Comment (jani270) [reply src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt]: > crazy rare chance, also do we know how that chat message looks like? just the gray x1?

image

Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/FossilExcavatorAPI.kt]: image

**Review by CalMWolfs — APPROVED** [review]

**Pull request merged**